source/agora/node/Ledger.d

```diff
@@ -518,6 +518,48 @@ public class Ledger
             return null;
         return block.getMerklePath(index);
     }
+
+    /***************************************************************************
+
+        Check if a transaction hash exists in the transaction pool.
+
+        Params:
+            tx = the transaction hash
+
+        Returns:
+            true if the transaction pool has the transaction hash.
+
+    ***************************************************************************/
+
+    public bool hasTransactionHash (const ref Hash tx) @safe
+    {
+        return this.pool.hasTransactionHash(tx);
```

Comment:
@linked0 Ping. This forwarding function does not need to exist.

Reply:
> This forwarding function does not need to exist.

I'll check it. Ty.